changelog.d/2037.change.rst

```diff
@@ -0,0 +1,4 @@
+A bit "cosmetic" changes though needed for implementing more advanced stuff.
+* Renamed args names params for some static methods and marked them with ''@staticmethod''.
```

Comment:
What's the purpose of this change? What value does it provide? What issue does it address?

Reply:
1. Warnings in #2034 about legacy calling convention (the arg is not named `dist`)
2. it turned out that the methods are static ones, but they look like bound ones. So I have marked them as static, no make this explicit and also allow static analysis tools to capture that.